teamcity/README.md

```diff
@@ -37,85 +37,52 @@ To configure this check for an Agent running on a host:
 
 Edit the `teamcity.d/conf.yaml` in the `conf.d/` folder at the root of your [Agent's configuration directory][3]. See the [sample teamcity.d/conf.yaml][4] for all available configuration options:
 
-The TeamCity check offers two methods of data collection. To optimally monitor your TeamCity environment, configure two separate instances to collect metrics from each method. 
+To collect metrics for your TeamCity environment, configure an instance using the TeamCity Server REST API method:
 
-1. OpenMetricsV2 method (requires Python version 3):
+Configure a separate instance in the `teamcity.d/conf.yaml` file to collect additional build-specific metrics, service checks, and build status events from the TeamCity server's REST API. Specify your projects and build configurations using the `projects` option (requires Python version 3):
 
-   Enable `use_openmetrics: true` to collect metrics from the TeamCity `/metrics` Prometheus endpoint.
 
+```yaml
+init_config:
 
-   ```yaml
-   init_config:
-   
-   instances:
-       ## @param server - string - required
-       ## Specify the server name of your TeamCity instance.
-       ## Enable Guest Authentication on your instance or enable the
-       ## optional `basic_http_authentication` config param to collect data.
-       ## If using `basic_http_authentication`, specify:
-       ##
-       ## server: http://<USER>:<PASSWORD>@teamcity.<ACCOUNT_NAME>.com
-       #
-     - server: http://teamcity.<ACCOUNT_NAME>.com
-       ## @param use_openmetrics - boolean - optional - default: false
-       ## Use the latest OpenMetrics V2 implementation to collect metrics from
-       ## the TeamCity server's Prometheus metrics endpoint.
-       ## Requires Python version 3.
-       ##
-       ## Enable in a separate instance to collect Prometheus metrics.
-       ## This option does not collect events, service checks, or metrics from the TeamCity REST API.
-       #
-       use_openmetrics: true
-   ```
-**Note:** To collect [OpenMetrics-compliant][16] histogram and summary metrics (available starting in TeamCity Server 2022.10+), add the internal property, `teamcity.metrics.followOpenMetricsSpec=true`. See, [TeamCity Internal Properties][25].
-
-2. TeamCity Server REST API method:
-
-   Configure a separate instance in the `teamcity.d/conf.yaml` file to collect additional build-specific metrics, service checks, and build status events from the TeamCity server's REST API. Specify your projects and build configurations using the `projects` option (requires Python version 3):
+instances:
+  - server: http://teamcity.<ACCOUNT_NAME>.com
 
```

Comment:
We still offer both methods of data collection, it's just that in the case of the OpenMetrics method, we don't need to refer to the V1 / V2 distinction because only V2 is available. This would also apply to the metadata.csv (in the same way that @iliakur has already commented).